docs/pod-metrics.md

```diff
@@ -11,6 +11,7 @@
 | kube_pod_labels | Gauge | Kubernetes labels converted to Prometheus labels                      | | `pod`=&lt;pod-name&gt; <br> `namespace`=&lt;pod-namespace&gt; <br> `label_POD_LABEL`=&lt;POD_LABEL&gt; <br> `uid`=&lt;pod-uid&gt; | STABLE | - |
 | kube_pod_nodeselectors| Gauge | Describes the Pod nodeSelectors                                       | |  `pod`=&lt;pod-name&gt; <br> `namespace`=&lt;pod-namespace&gt; <br> `nodeselector_NODE_SELECTOR`=&lt;NODE_SELECTOR&gt; <br> `uid`=&lt;pod-uid&gt; | EXPERIMENTAL | Opt-in |
 | kube_pod_status_phase | Gauge | The pods current phase                                                | | `pod`=&lt;pod-name&gt; <br> `namespace`=&lt;pod-namespace&gt; <br> `phase`=&lt;Pending\|Running\|Succeeded\|Failed\|Unknown&gt; <br> `uid`=&lt;pod-uid&gt; | STABLE | - |
+| kube_pod_status_qos_class | Gauge | The pods current qosClass                                                | | `pod`=&lt;pod-name&gt; <br> `namespace`=&lt;pod-namespace&gt; <br> `qos_class`=&lt;BestEffort\|Burstable\|Guaranteed&gt; <br> `uid`=&lt;pod-uid&gt; | STABLE | - |
```

Comment:
Please make those metrics experimental first when introducing them.

Reply:
Yes, you can use this function
```
		*generator.NewFamilyGeneratorWithStability(
            ...,
			basemetrics.ALPHA,
            ...,
        )
```